tools/stats/grouping.py

```diff
@@ -121,10 +121,9 @@ def main():
 
     def is_new_item(line):
         try:
-            item = line.split("\t")[group_col]
+            item = line.rstrip("\r\n").split("\t")[group_col]
```

Comment:
`rstrip("\r\n")` will chop off any number of trailing newlines ... but I guess this is probably ok.

Reply:
but I think we should bump the version number.

Reply:
This is already operating on a single line, I don't think there can be more than one "\r\n".